reflex/components/media/image.py

```diff
@@ -60,9 +63,37 @@ def get_triggers(self) -> Set[str]:
         return super().get_triggers() | {"on_error", "on_load"}
 
     def _render(self) -> Tag:
-        # If the src is an image, convert it to a base64 string.
-        if types.is_image(type(self.src)):
-            self.src = Var.create(format.format_image_data(self.src))  # type: ignore
+        try:
+            from PIL.Image import Image as Img
+
+            # If the src is an image, convert it to a base64 string.
+            if types._isinstance(self.src, Img):
+                self.src = Var.create(serialize(self.src))  # type: ignore
+        except ImportError:
+            pass
```

Comment:
Do we need the import guard here? Doesn't seem like this block is actually using `Img`

Reply:
i guess i was thinking we can just call `self.src = Var.create(serialize(self.src))` and not care about what `self.src` is... looks like `serialize` has a default `str` handler.